# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-3rd-party-crates.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-139.0.7258.127/build/rust/cargo_crate.gni.vanilla	2025-08-13 20:20:02.271946855 +0200
+++ chromium-139.0.7258.127/build/rust/cargo_crate.gni	2025-08-13 20:20:29.402453035 +0200
@@ -310,6 +310,8 @@
       rustflags +=
           [ "-Awarnings" ]  # Suppress other warnings in 3rd-party crates.
 
+      rustflags += [ "-Amismatched_lifetime_syntaxes" ]
+
       if (!defined(build_native_rust_unit_tests)) {
         build_native_rust_unit_tests = _crate_type != "proc-macro"
       }
